src/NuGet/Microsoft.Net.Compilers.Toolset/AnyCpu/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -61,10 +62,12 @@
       <TfmSpecificPackageFile Include="@(_FileWithPath)" PackagePath="%(_FileWithPath.TargetPath)" />
     </ItemGroup>
 
-    <!-- Used for debugging parts of the toolset (e.g., build tasks). -->
+    <!-- Used for debugging parts of the toolset (e.g., build tasks).
+      Doesn't work in VS (where `<ProjectReference Targets="Publish" />` doesn't have any effect), hence we set `ContinueOnError="true"`. -->
     <Copy SourceFiles="@(_FileWithPath)"
           DestinationFiles="$(BaseOutputPath)$(Configuration)\%(_FileWithPath.TargetPath)"
-          SkipUnchangedFiles="true" />
+          SkipUnchangedFiles="true"
+          ContinueOnError="true" />
```

Comment:
Should we instead condition on `Condition="'$(DesignTimeBuild)' != 'true'"` - or does it actually need to run?

Reply:
Ah we don't want this running in design time builds either. That AfterTargets="Build" really needs to go since that's still going to cause issues here. I'm going to make another push to this PR.

Reply:
`Condition="'$(DesignTimeBuild)' != 'true'"` sounds like a good idea. I'm not sure why `AfterTargets="Build"` is bad (in this scenario), but feel free to improve.

Reply:
> why AfterTargets="Build" is bad

It's easier to delete it than it is to convince ourselves it's correct in all situations. :smile: The problem is there's enough special types of builds that do strange things with project references that it's just not really safe to make the assumption. For example, you can also do a command line build passing a property to not build dependencies, and I'm not sure what that would do.

So yes, we could add a condition to remove DesignTimeBuilds, but if we just delete the AfterTargets="Build" we dont' have to do anything in the first place. Less is more.

Reply:
There is one potential workflow change if you did like the ability to just do a build and see the files; that said we're adding I/O for copies for the 99.9% of the time people don't need that. If we wanted to refactor this a bit we could have an explicit target do the copy and you can do that.

Reply:
It's not just that I like the workflow, I've added it so it can be used for debugging, e.g.:

https://github.com/dotnet/roslyn/blob/489d016f12bc69c4ace6f5015f27a9387bc4697d/.vscode/launch.json#L103

That being said, there could probably be a property to opt in to it when needed.